src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -330,6 +330,24 @@ public static boolean isOlderThanAYear(Date compareDate) {
     }
     
     /**
+     * returns true if the session is closed within the past hour of calling this function
```

Comment:
Also, do not mention anything about "session" here. This is a general utility function that is independent of call site.